setup.py

```diff
@@ -55,7 +55,9 @@ def get_advanced_templates():
     entry_points={
         'console_scripts': [
             'release=release:main',
-            'ccm-deploy-test=test_util.test_installer_ccm:main',
+            # Remove this once CI is updated
+            'ccm-deploy-test=test_util.test_aws_vpc:main',
```

Comment:
can we just keep using the old name for now rather than make it so the CI doesn't work on old commits / old builds? Eventually we should move all of these to some common wrapper where we can dispatch to the individual variations with flags rather than one binary per test shape